packages/components/text/__tests__/text.test.tsx

```diff
@@ -5,43 +5,56 @@ import Text from '../src/text.vue'
 const AXIOM = 'Rem is the best girl'
 
 describe('Text.vue', () => {
-  test('render text & class', () => {
-    const wrapper = mount(() => (
+  test('create', () => {
+    const wrapper = mount(
       <Text
         v-slots={{
           default: () => AXIOM,
         }}
       />
-    ))
-    const vm = wrapper.vm
+    )
 
-    expect(vm.$el.classList.contains('el-text')).toEqual(true)
-    expect(wrapper.text()).toEqual(AXIOM)
+    expect(wrapper.classes()).toContain('el-text')
   })
 
   test('type', () => {
-    const wrapper = mount(() => <Text type="success" />)
-    const vm = wrapper.vm
-    expect(vm.$el.classList.contains('el-text--success')).toEqual(true)
+    const wrapper = mount(<Text type="success" />)
```

Comment:
## Suggest

Can you change back to the old way? Most of the test code follows the method below.

```vue
mount(() => <Text />)
```

Reply:
ok, go home from get off work and continue.